apps/stats/management/commands/__init__.py

```diff
@@ -92,3 +101,24 @@ def get_date_from_file(filepath, sep):
             return line.split(sep)[0]
         except IndexError:
             return None
+
+
+def stats_from_model(model):
+    """Return the stats from the model ready to write to a file."""
+    data = model_to_dict(model)
+    del data['date']  # Already in the filename.
+    del data['addon']  # Already in the file's folder name.
+    del data['id']  # No need for the model's ID at all (eg: UpdateCount).
+    return json.dumps(data)
```

Comment:
Sure it's a bit redundant but I feel like information should be clustered together rather than obtained from different locations.